app/src/main/java/io/actifit/fitnesstracker/actifitfitnesstracker/MainActivity.java

```diff
@@ -2273,7 +2447,7 @@ private void checkBatteryOptimization(Boolean forceShow) {
 
         PowerManager pm = (PowerManager) getSystemService(Context.POWER_SERVICE);
         FontTextView batteryNotif = findViewById(R.id.battery_notice);
-        if (!pm.isIgnoringBatteryOptimizations("io.actifit.fitnesstracker.actifitfitnesstracker")) {
+        if (!pm.isIgnoringBatteryOptimizations("com.example.trial")) {

```

Comment:
@AliHajjAli1 this is not the correct package name, seems you overwrote the original one "io.actifit.fitnesstracker.actifitfitnesstracke"